utils/mock-data-loader/banks/index.js

```diff
@@ -20,6 +20,17 @@ const BANKS = [
     companiesHouseNo: '00014259',
     partyUrn: '00300342',
   },
+  {
+    id: '999',
+    name: 'GEF Only Bank',
+    mga: ['UKEF-GEF-only-amended-and-restated-MGA-dated-31-January-2020.pdf'],
+    emails: [
+      'checker4@ukexportfinance.gov.uk',
```

Comment:
Can we check with Caroline what email she'd like to use for this bank

Reply:
Caroline is fine with this email address